packages/api/src/services/plugins/plugins-rest.service.ts

```diff
@@ -17,7 +16,7 @@ export class PluginsRestService extends HttpService {
    * information about available plugins.
    */
   getPluginsManifest(): Promise<PluginsManifestResponse> {
-    return this.get(MANIFEST_URL);
+    return this.get(ServiceProvider.get(ConfigurationContextService).getApplicationConfiguration().pluginsManifestPath);
```

Comment:
IMO, this method should not be here because it does not call the BE.
I think it would be better if we map the configuration response in `ConfigurationService.getConfiguration` and update the `applicationConfiguration` of `ConfigurationContextService` with the UI model.

The configuration mapper function can use a concrete mapper for a specific configuration (for now: `pluginsManifestMapper.mapToModel`).

Then, in the code, we can subscribe to or synchronously get the configuration through the context. In that case, the `getPluginsManifest` method in `PluginsService` can either be removed or re-implemented to use `ConfigurationContextService` with mapped data, instead of mapping it every time it is called.

Reply:
This comment is invalid. The getPluginsManifest method retrieves the `pluginsManifestPath` from the configuration context and then makes a GET request to fetch the plugins manifest JSON file from that location.

Reply:
My, fault. I missed `this.get`, which is the REST call. Ignore the comment..